content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -36,84 +49,248 @@ This section of the vehicle form relates to the person who is currently driving
+form, add any other contacts and addresses associated with the driver or company in this tab.
+
+To add a new contact, click the :guilabel:`Add` button, and a :guilabel:`Create Contact` pop-up
+window appears.
+
+Enter the following information on the form:
+
+- :guilabel:`Contact Type`: select the type of contact being added. Click the radio button next to
+  the desired type. The default options are:
+
+  - :guilabel:`Contact`: select this option to add general contact details for employees of the
+    associated company.
+  - :guilabel:`Invoice Address`: select this option to add a preferred address for all invoices.
+    When added to the form, this address is selected by default when sending an invoice to the
+    associated company.
+  - :guilabel:`Delivery Address`: select this option to add a preferred address for all deliveries.
+    When added to the form, this address is selected by default when delivering an order to the
+    associated company.
+  - :guilabel:`Follow-up Address`: select this option to add a preferred address for all follow-up
+    correspondence. When added to the form, this address is selected by default when sending
+    reminders about overdue invoices.
+  - :guilabel:`Other Address`: select this option to add any other necessary addresses for the
+    company.
+
+.. image:: new_vehicle/create-contact.png
+   :align: center
+   :alt: The create contact form with all parts filled in.
+
+Depending on the :guilabel:`Contact Type`, some optional fields may not be visible. The available
+fields are identical to the fields in the :ref:`general information
+<fleet/new_vehicle/general-info>` section of the new driver form.
+
+Add any notes to the :guilabel:`Internal notes...` section of the form.
+
+After entering all of the information, click either :guilabel:`Save & Close` to add the one new
+contact, or :guilabel:`Save & New` to add the current address record and create another address
+record.
+
+As contacts are added to this tab, each contact appears in a separate box, with an icon indicating
+what type of contact is listed.
+
+.. example::
+   An :guilabel:`Invoice Address` displays a dollar sign icon inside that specific address box,
+   whereas a :guilabel:`Delivery Address` displays a truck icon inside.
+
+   .. image:: new_vehicle/contacts-address.png
+     :align: center
+     :alt: The create contact form with all parts filled in.
+
+Sales & purchase tab
+********************
+
+Enter the following sales and purchasing information for the various sections below. Depending on
+the other installed applications, additional fields and sections may appear. The following are all
+default fields for the *Fleet* application only.
+
+Sales
+^^^^^
+
+- :guilabel:`Salesperson`: using the drop-down menu, select the user who is the main point of
+  contact for sales with this driver's company.
+
+  This person **must** be an internal user of the company, meaning they can log into the database as
+  a user.
+
```

Comment:
I'm also seeing fields for: Payment Terms, Pricelist, Avalara Code, Delivery Method...

...so please double-check to ensure all the necessary fields are being mentioned/explained. Thanks!

Reply:
I had a soul-searching moment with this, because you are correct- in various Runbots, with ALL the apps installed, there are a lot of other fields and tabs that are present. So after first making a VERY long (too long in my opinion) version of this section, I then made a DB to see what was actually in a Fleet install without other items. So I went with that route- and why I had added that 'this is only the defaults for fleet' statement.